common/src/main/resources/messages/messages.properties

```diff
@@ -171,6 +171,15 @@ cluster.externaldatabase.stop.commenced=External database stop commenced.
 cluster.externaldatabase.stop.failed=External database stop failed.
 cluster.externaldatabase.stop.finished=External database stop finished.
 
+
+cluster.ccm.upgrade.preparation=CCM upgrade preparation started.
+cluster.ccm.upgrade.reregister=CCM upgrade re-registration started.
+cluster.ccm.upgrade.remove.autossh=CCM upgrade autossh remove.
+cluster.ccm.upgrade.unregister.hosts=CCM upgrade unregister hosts.
```

Comment:
`CCM upgrade re-registration started.` perhaps change it to `CCM upgrade re-register hosts.` to be in sync with `cluster.ccm.upgrade.unregister.hosts`.